src/Moryx.CommandCenter.Web/src/common/scss/commandcenter.scss

```diff
@@ -9,500 +9,566 @@
+            min-width: 100px;
+        }
     }
-  }
 }
 
+.toggle-on {
+    min-width: 62px;
+}
+
+.card>.list-group {
+    border-bottom: none;
+}
+
+.list-group-item.selectable {
+    padding: .75rem 1.25rem;
+    border: 1px solid rgba(0, 0, 0, 0.125);
+
+}
+
+.list-group-item.selectable:not(:first-child) {
+    border-top: none;
+}
 
 /*
  * Sidebar
  */
 .sidebar-panel {
-  .panel-body {
-    padding: 0px;
-  }
-  
-  .nav-wrapper {
-    position: relative;
-    min-height: 50px;
-  }
-  
-  .navlist {
-    padding-left: 0;
-    margin-bottom: 0;
-    list-style: none;
-  }
-  
-  .navitem > a, .nav-subitem {
-    position: relative;
-    display: block;
-    cursor: pointer;
+    .panel-body {
+        padding: 0px;
+    }
 
-    &:hover, &.active {
-      background-color: #eee;
+    .nav-wrapper {
+        position: relative;
+        min-height: 50px;
     }
-    
-    .navdropdown {
-      padding-left: 20px;
+
+    .navlist {
+        padding-left: 0;
+        margin-bottom: 0;
+        list-style: none;
+    }
+
+    .navitem>a,
+    .nav-subitem {
+        position: relative;
+        display: block;
+        cursor: pointer;
+
+        &:hover,
+        &.active {
+            background-color: #eee;
+        }
+
+        .navdropdown {
+            padding-left: 20px;
+        }
+    }
+
+    .navitem>a,
+    .nav-subitem>a {
+        //color: $linkColor;
+        position: relative;
+        display: block;
+        padding: 10px 10px;
+        text-decoration: none;
     }
-  }  
-  
-  .navitem>a, .nav-subitem>a{
-    //color: $linkColor;
-    position: relative;
-    display: block;
-    padding: 10px 10px;
-    text-decoration: none;
-  }
 }
 
 /*
  * Modals
  */
- .modal {
-  text-align: center;
-  padding: 0!important;
+.modal {
+    text-align: center;
+    padding: 0 !important;
 }
 
 .modal:before {
-  content: '';
-  display: inline-block;
-  height: 100%;
-  vertical-align: middle;
-  margin-right: -4px;
+    content: '';
+    display: inline-block;
+    height: 100%;
+    vertical-align: middle;
+    margin-right: -4px;
 }
 
 .modal-dialog {
-  display: inline-block;
-  text-align: left;
-  vertical-align: middle;
+    display: inline-block;
+    text-align: left;
+    vertical-align: middle;
 }
 
 /*
  * Font
  */
 .font-bold {
-  font-weight: bold;
+    font-weight: bold;
 }
 
 .font-italic {
-  font-style: italic;
+    font-style: italic;
 }
 
 .font-disabled {
-  color: gray;
+    color: gray;
 }
 
 .font-normal {
-  font-size: 16px;
+    font-size: 16px;
 }
 
 .font-small {
-  font-size: 14px;
+    font-size: 14px;
 }
 
 .font-smaller {
-  font-size: 12px;
+    font-size: 12px;
 }
 
 .font-smallest {
-  font-size: 10px;
+    font-size: 10px;
 }
 
 /*
  * Progress
  */
 .progress {
-  height: 20px;
-  background: whitesmoke;
+    height: 20px;
+    background: whitesmoke;
 }
 
 .progress-bar {
-  background: orangered;
-  color: black;
+    background: orangered;
+    color: black;
 }
 
 /*
```

Comment:
Removed the footer classes but kept the modal-dialog because it is still being used [here](https://github.com/jsonBackup/MORYX-Framework/blob/51c1e7e3746dc879255bb3a733535552ec80648e/src/Moryx.CommandCenter.Web/src/modules/container/Module.tsx#L259)